services/utils.py

```diff
@@ -0,0 +1,19 @@
+from services.models import ServiceClientId
+
+
+def set_service_to_request(request):
```

Comment:
Could this be a middleware?

Reply:
There are currently two JWT based authentication systems in this project, an [older one](https://github.com/City-of-Helsinki/open-city-profile/blob/HP-575-service-from-azp-claim/open_city_profile/oidc.py#L12-L30) and a [newer one](https://github.com/City-of-Helsinki/open-city-profile/blob/HP-575-service-from-azp-claim/open_city_profile/middleware.py#L8-L29). Only one or the other can be in use, there's a [feature flag](https://github.com/City-of-Helsinki/open-city-profile/blob/HP-575-service-from-azp-claim/open_city_profile/settings.py#L67) to toggle which one.

This helper is called from both of those two authentication systems. Once we get rid of the older authentication system, then this helper could be merged into the newer authentication middleware, if it feels like the right thing to do.